python.html.markdown

```diff
@@ -328,7 +328,7 @@ filled_dict["four"] = 4  # now, filled_dict["four"] => 4
 filled_dict.setdefault("five", 5)  # filled_dict["five"] is set to 5
 filled_dict.setdefault("five", 6)  # filled_dict["five"] is still 5
 
-# Sets store ... well sets (which are like lists but can contain no duplicates)
+# You can declare sets (which are like unordered lists that cannot contain duplicated values) using the set object.
```

Comment:
Please change this to 
```
# You can declare sets (which are like unordered lists that cannot contain
# duplicate values) using the set object.
```

This restricts line length to 80 chars as required inside code blocks and fixes 'duplicated' to 'duplicate'.

Reply:
Done! Thank you @divayprakash :)